sogs/templates/base.html

```diff
@@ -1,9 +1,79 @@
-<html>
+<html id="html" class="dark">
   <head>
+    
     <title> sogs </title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="{{ url_for('static', filename='output.css') }}">
   </head>
-  <body>
+  <body class="dark:bg-otherGray transition duration-500 h-full flex flex-col overflow-hidden">
+    
+    {% block modal %}
+    {% endblock %}
+    
+    <!-- header / navbar -->
+    <div class="px-2 sm:px-5 flex-none flex justify-around sm:justify-between items-center w-full h-32 max-h-32 
+    bg-white dark:bg-otherGray border-b-2 border-black dark:border-lightGray transition duration-500">
+      <div class="text-black text-center">
+        <button>
+          <a href="/">
+            <h1 class="  dark:text-white text-3xl xs:text-4xl sm:text-5xl decoration-black dark:decoration-white decoration-2">{% block heading %}{% endblock %}</h1>
+            <div class="flex flex-col sm:flex-row pl-3 sm:pl-0">
+              <p class=" font-bold text-sessionGreen dark:text-sessionGreen text-lg xs:text-xl sm:text-2xl">Session Open&nbsp</p>
+              <p class=" font-bold text-sessionGreen dark:text-sessionGreen text-lg xs:text-xl sm:text-2xl">Group Server</p>
+            </div>
+          </a>
+        </button>
+      </div>
+      
+      <div class="flex space-x-4 ">
+        {% block invite %}
+        {% endblock %}
+        <button id="moonIcon" class='px-1 rounded' onclick="toggleDarkMode()"><img src="{{ url_for('static', filename='moon-d-50-min.png') }}" alt="moon"></button>
```

Comment:
can you use svg instead

Reply:
no SVGs please.

Reply:
why not?
Is still related to security risks that svgs can run JS code? If yes, better not use JS at all, right? If a user can change the svg file to include malicious JS code, surely he can change the JS code itself to make it malicious, right?